jetty-core/jetty-start/pom.xml

```
@@ -8,7 +8,7 @@
   </parent>
   <artifactId>jetty-start</artifactId>
   <name>Core :: Start</name>
-  <description>The start utility</description>
+  <description>Jetty startup and bootstrap module.</description>
```

It's not a generic startup / bootstrap, it belongs specifically to jetty-home.

```suggestion
  <description>Jetty Home Start Module.</description>
```